web3.go

```diff
@@ -46,7 +46,7 @@ var Networks = map[string]Network{
 		Unit: "GO",
 	},
 	"ethereum": {
-		URL:  "https://main-rpc.linkpool.io",
+		URL:  "https://cloudflare-eth.com",
```

Comment:
Is cloudflare more stable?

Reply:
linkpool wasn't working when I tried sending the transactions. Didn't look into it much, but cloudflare worked right away so... 🤷‍♂️